### ISSUE TITLE
failed to match fitting object

### ISSUE THREAD

**Issue by matkoniecz** [EdwardBetts/osm-wikidata#150]
https://osm.wikidata.link/Q11721219 and https://osm.wikidata.link/Q1013422 failed to match despite nearby matching objects present (or maybe I missed some necessary criteria)

https://osm.wikidata.link/Q11721219 should match to https://www.openstreetmap.org/node/4002924424#map=19/50.21911/19.79613 and https://www.openstreetmap.org/node/4002924432

https://osm.wikidata.link/Q1013422 should match to https://www.openstreetmap.org/way/120331996#map=19/50.21193/19.83071

**Comment by EdwardBetts** [comment]
Exactly.

**Comment by matkoniecz** [comment]
Hmm, maybe it would be useful to mention it on no_match page so people will know how it may be improved?

> Items may not match as result of missing data about names in OpenStreetMap and Wikidata. This may be fixed by adding missing names in OSM (name, alt_name, short_name, loc_name, official_name tags may be used) and labels on WIkidata and refreshing cache.

**Comment by EdwardBetts** [comment]
Yes, I should add some text to explain. The item detail page might also list nearby items where the tag matches but the name doesn't match.
